notification/src/handler/notification/notification.handler.js

```diff
@@ -270,6 +281,35 @@ function getChangeTransactionStateUpdateAction(
   }
 }
 
+function convertDateToUTCFormat(transactionEventDate, logger) {
+  try {
+    // Assume transactionEventDate should be in correct format (e.g. '2019-01-30T18:16:22+01:00')
+    const eventDateMilliSecondsStr = Date.parse(transactionEventDate)
+    const transactionDate = new Date()
```

Comment:
```suggestion
    const transactionDate = new Date(eventDateMilliSecondsStr)
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#parameters

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/da4e9bc8a0ac09719dbc657d3d9fee96b4d09a38